packages/api/lib/testUtils.js

```diff
@@ -240,7 +241,19 @@ function fakeAccessTokenFactory(params = {}) {
   };
 }
 
+async function createJwtAuthToken({ accessTokenModel, userModel }) {
+  const userRecord = fakeUserFactory();
+  await userModel.create(userRecord);
+
+  const accessTokenRecord = fakeAccessTokenFactory({ username: userRecord.userName });
+  await accessTokenModel.create(accessTokenRecord);
+
+  return createJwtToken(accessTokenRecord);
+}
+
 module.exports = {
+  createJwtAuthToken,
+  createJwtToken,
```

Comment:
But a new thing, why is this exported from here?

Reply:
No good reason